hibike/devices/YogiBear/LED.cpp

```diff
@@ -28,6 +31,9 @@ void ctrl_YELLOW() {
 	} else {
 		digitalWrite(LED_YELLOW, LOW);
 	}
+	if(!isMotorEnabled()) {
```

Comment:
Okay so at this point, we're duplicating code.
Change all the led funtions to be a single function, that takes an additional arg: COLOR

you already have `LED_GREEN` etc. as variables: pass that in as the argument.